### ISSUE TITLE
Buildkite producing invalid terraform plan

### ISSUE THREAD

**Issue by pink** [buildkite/terraform-provider-buildkite#507]
This seems to be similar to the error seen in https://github.com/buildkite/terraform-provider-buildkite/issues/502

I've been trying to run a Terraform apply for our org and have been hitting multiple instances of this error:
```
╷
│ Error: Provider produced invalid plan
│ 
│ Provider "registry.terraform.io/buildkite/buildkite" planned an invalid value for buildkite_pipeline.website.steps: planned value cty.StringVal("steps:\n- label:
│ ':pipeline: Pipeline Upload'\n  command: buildkite-agent pipeline upload") does not match config value cty.UnknownVal(cty.String).
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
╵
```

We're on version 1.4.1 of the Buildkite terraform provider.

**Comment by mcncl** [comment]
@pink thanks for raising this. I've asked @jradtilbrook to follow up with some information on how this was resolved in the mentioned issue.

**Comment by jradtilbrook** [comment]
Ahh yes. The case with #502 was more of a misconfiguration of terraform. Can you share with us a minimal reproduction where you see this issue?
Feel free to email us at support@buildkite.com if you want to send the exact info and keep it private.

**Comment by jradtilbrook** [comment]
Thanks for reporting this @pink. It should be resolved with #539
